custom_model_runner/datarobot_drum/drum/runtime.py

```diff
@@ -42,15 +42,7 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
             # exception occurred before args were parsed
             return False  # propagate exception further
 
-        msg = "Looks like there is a problem."
-        if not self.options.verbose:
-            msg += f" To get more output information try to run locally(not in DataRobot) with: '{ArgumentsOptions.VERBOSE}'."
-        logger_drum.warning(colored(msg, "yellow"))
-
-        if exc_value:
-            logger_drum.error(exc_value)
-        logger_drum.error(exc_type)
-        logger_drum.error("".join(traceback.format_list(traceback.extract_tb(exc_traceback))))
```

Comment:
If you remove line seems like `exc_traceback` will not get printed, or would it get printed somewhere else?

Reply:
it will be printed because I explicitly call logger's `exception`  method, which takes `message` as an argument, which I provide as empty.